virtool/references/data.py

```diff
@@ -982,3 +985,14 @@ async def clean_all(self):
                         {"$pop": {"updates": -1}, "$set": {"updating": False}},
                         session=session,
                     )
+
+    async def update_references(self):
```

Comment:
```suggestion
    async def fetch_and_update_reference_releases(self):
```
The name as written suggests that the function will update the references themselves, rather than simply updating the  list of known remote releases